corecomponent/androidcomponent/src/main/res/menu/menu_nav_drawer.xml

```diff
@@ -7,7 +7,7 @@
         >
         <item
             android:id="@id/main"
-            android:icon="@drawable/ic_event_outline_black_24dp"
+            android:icon="@drawable/ic_event_24px"
```

Comment:
Drawable file's name referred to Figma's content.
https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=21%3A543

Should I rename other drawable file's name?
ex. `ic_android_black_24dp.xml` -> `ic_android_24dp.xml`